src/System.Net.Requests/tests/HttpWebRequestTest.netstandard1.7.cs

```diff
@@ -0,0 +1,28 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
@weshaggard is this how we envision testing for all future versions? Each new feature will need to be separated into a new versioned file?